openlibrary/solr/update_work.py

```diff
@@ -342,6 +359,37 @@ def process_editions(self, w, editions, ia_metadata, identifiers):
                             identifiers[k].append(v)
         return sorted(editions, key=lambda e: int(e.get('pub_year') or -sys.maxsize))
 
+    @staticmethod
+    def normalize_authors(authors) -> List[dict]:
+        """
+        Need to normalize to a predictable format because of inconsitencies in data
+
+        >>> SolrProcessor.normalize_authors([
+        ...     {'type': {'key': '/type/author_role'}, 'author': '/authors/OL1A'}
+        ... ])
+        [{'type': {'key': '/type/author_role'}, 'author': {'key': '/authors/OL1A'}}]
+        >>> SolrProcessor.normalize_authors([{
+        ...     "type": {"key": "/type/author_role"},
+        ...     "author": {"key": "/authors/OL1A"}
+        ... }])
+        [{'type': {'key': '/type/author_role'}, 'author': {'key': '/authors/OL1A'}}]
+        """
+        return [
+            {
+                'type': {
+                    'key': safeget(lambda: a['type']['key']) or '/type/author_role'
+                },
+                'author': (
+                    a['author'] if isinstance(a['author'], dict)
+                    else {'key': a['author']}
+                )
+            }
+            for a in authors
+            # TODO: Remove after
+            #  https://github.com/internetarchive/openlibrary-client/issues/126
```

Comment:
Is this referenced anywhere in e.g. #126 or elsewhere where we'll actually revisit?
```
# TODO: Remove after
#  https://github.com/internetarchive/openlibrary-client/issues/126
```